components/postmark/postmark.app.mjs

```diff
@@ -1,11 +1,71 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "postmark",
-  propDefinitions: {},
+  propDefinitions: {
+    templateAlias: {
+      type: "string",
+      label: "Template",
+      description: "The template to use for this email.",
+      async options(context) {
+        const { page } = context;
+        return this.listTemplates(page);
+      },
+    },
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _apikey() {
+      return this.$auth.api_key;
+    },
+    async listTemplates(page) {
+      const amountPerPage = 3;
+      const offset = page * amountPerPage;
+
+      const data = await axios(this, {
+        url: `https://api.postmarkapp.com/templates?Count=${amountPerPage}&Offset=${offset}&TemplateType=Standard`,
+        method: "GET",
+        headers: this.getHeaders(),
+      });
+
+      return data.TotalCount
+        ? data.Templates.map((obj) => {
+          return {
+            label: obj.Name,
+            value: obj.Alias,
+          };
+        })
+        : [];
+
```

Comment:
Yes, that makes sense, the `options`-specific object should be created in the `options` method. Adjusting to reflect that.

I was working with `prevContext` initially instead of `page`, and thought it was not needed to return that when using `page`. Adjusting the return value as you suggested.